src/tape/ensemble.py

```diff
@@ -1094,6 +1094,59 @@ def from_source_dict(self, source_dict, column_mapper=None, npartitions=1, **kwa
         self._object_dirty = False
         return self
 
+    def convert_flux_to_mag(self, flux_col, zero_point, err_col=None, zp_form="mag", out_col_name=None):
+        """Converts a flux column into a magnitude.
+
+        Parameters
+        ----------
+        flux_col: 'str'
+            The name of the ensemble flux column to convert into magnitudes.
+        zero_point: 'str'
+            The name of the ensemble column containing the zero point
+            information for column transformation.
+        err_col: 'str', optional
+            The name of the ensemble column containing the errors to propagate.
+        zp_form: `str`
+            The form of the zero point column, either "flux" or
+            "magnitude"/"mag". Determines how the zero point (zp) is applied in
+            the conversion. If "flux", then the function is applied as
+            Mag=-2.5*log10(flux/zp), or if "magnitude", then
+            Mag=-2.5*log10(flux)+zp.
+        out_col_name: 'str', optional
+            The name of the output magnitude column, if None then the output
+            is just the flux column name + "_mag". The error column is also
+            generated as the out_col_name + "_err".
+
+        Returns
+        ----------
+        ensemble: `tape.ensemble.Ensemble`
+            The ensemble object with a new mag column.
+
+        """
+        if out_col_name is None:
+            out_col_name = flux_col + "_mag"
+
+        if zp_form == "flux":  # mag = -2.5*np.log10(flux/zp)
```

Comment:
mag -> Mag to be consistent with the docstring (or change in docstring); just to be consistent

Reply:
I changed the "Mag" in the equation provided in the docstring instead.